juhwan/hello-world/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import reportWebVitals from './reportWebVitals';
+import {Hello} from "./components/Hello";
+import {Menu} from "./components/Menu";
```

Comment:
Hello와 Menu 컴포넌트를 작성할 때, 단일 함수형 컴포넌트만을 작성하였기 때문에 해당 라인에서 import할 떄 중괄호를 사용하는 것은 불필요한 코드입니다. 다만 default 키워드가 누락되어 불필요하게 중괄호를 사용하여 import문이 작성되어 있습니다.
```suggestion
import Hello from "./components/Hello";
import Menu from "./components/Menu";
```
아래 블로그 내용을 참고해보시길 바랍니다.
https://resilient-923.tistory.com/305